### PR TITLE
Including guidance for Windows redirection GPO

### PR DESCRIPTION
**Pull request by cartergreen**
We have been receiving feedback from many customers that they are lacking this specific guidance.  In customer meetings we have reviewed this guidance and it has helped, so I'm proposing that we include this guidance on our support page for admins.

### CODE REVIEW

**Comment by cartergreen** [comment]
@kaarins 
@gacarini

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: OneDrive/redirect-known-folders.md [hunk 1]
```diff
@@ -87,3 +87,25 @@ To use the following Group Policy objects, you need the OneDrive sync build 18.1
 - [Prevent users from moving their Windows known folders to OneDrive](use-group-policy.md#BlockKnownFolderMove)
 
 For info about using the OneDrive Group Policy objects, see [Use Group Policy to control OneDrive sync app settings](use-group-policy.md).
+
+## Moving from previous Windows Folder Redirection Group Policy
+The OneDrive Known Folder Move Group Policy objects won't work if you previously used Windows Folder Redirection Group Policy objects to redirect the Documents, Pictures, or Desktop folders to a location other than OneDrive. The OneDrive Group Policy objects won't affect the Music and Videos folders, so you can keep them redirected with the Windows Group Policy objects. Here are the steps you must take to get into a state where the Known Folder Move Group Policy will work:
+- If folder(s) have been redirected to OneDrive using Windows Folder Redirection Group Policy
```
Comment (JohanFreelancer9) [inline OneDrive/redirect-known-folders.md:93]:
```suggestion
- If folder(s) have been redirected to OneDrive using Windows Folder Redirection Group Policy;
```

File: OneDrive/redirect-known-folders.md [hunk 2]
```diff
@@ -87,3 +87,25 @@ To use the following Group Policy objects, you need the OneDrive sync build 18.1
...
+  1. Disable the Window Folder Redirection Group Policy, make sure to leave the folder and contents on OneDrive
```
Comment (JohanFreelancer9) [inline OneDrive/redirect-known-folders.md:94]:
```suggestion
  1. Disable the Window Folder Redirection Group Policy and make sure to leave the folder and contents on OneDrive.
```

File: OneDrive/redirect-known-folders.md [hunk 3]
```diff
@@ -87,3 +87,25 @@ To use the following Group Policy objects, you need the OneDrive sync build 18.1
...
+  2. Enable KFM Group Policy
```
Comment (JohanFreelancer9) [inline OneDrive/redirect-known-folders.md:95]:
```suggestion
  2. Enable KFM Group Policy.
```

File: OneDrive/redirect-known-folders.md [hunk 4]
```diff
@@ -87,3 +87,25 @@ To use the following Group Policy objects, you need the OneDrive sync build 18.1
...
+  3. Your known folders shouldn't have moved anywhere
```
Comment (JohanFreelancer9) [inline OneDrive/redirect-known-folders.md:96]:
```suggestion
  3. Your known folders shouldn't have moved anywhere.
```

File: OneDrive/redirect-known-folders.md [hunk 5]
```diff
@@ -87,3 +87,25 @@ To use the following Group Policy objects, you need the OneDrive sync build 18.1
...
+- If folder(s) have been redirect to location on local PC
```
Comment (JohanFreelancer9) [inline OneDrive/redirect-known-folders.md:97]:
```suggestion
- If folder(s) have been redirected to a location on a local PC;
```

File: OneDrive/redirect-known-folders.md [hunk 6]
```diff
@@ -87,3 +87,25 @@ To use the following Group Policy objects, you need the OneDrive sync build 18.1
...
 For info about using the OneDrive Group Policy objects, see [Use Group Policy to control OneDrive sync app settings](use-group-policy.md).
+
+## Moving from previous Windows Folder Redirection Group Policy
+The OneDrive Known Folder Move Group Policy objects won't work if you previously used Windows Folder Redirection Group Policy objects to redirect the Documents, Pictures, or Desktop folders to a location other than OneDrive. The OneDrive Group Policy objects won't affect the Music and Videos folders, so you can keep them redirected with the Windows Group Policy objects. Here are the steps you must take to get into a state where the Known Folder Move Group Policy will work:
+- If folder(s) have been redirected to OneDrive using Windows Folder Redirection Group Policy
+  1. Disable the Window Folder Redirection Group Policy, make sure to leave the folder and contents on OneDrive
+  2. Enable KFM Group Policy
+  3. Your known folders shouldn't have moved anywhere
+- If folder(s) have been redirect to location on local PC 
+  1. Disable the Window Folder Redirection Group Policy, make sure to redirect the folder(s) back to the user profile location
```
Comment (JohanFreelancer9) [inline OneDrive/redirect-known-folders.md:98]:
```suggestion
  1. Disable the Window Folder Redirection Group Policy and make sure to redirect the folder(s) back to the user profile location.
```

File: OneDrive/redirect-known-folders.md [hunk 7]
```diff
@@ -87,3 +87,25 @@ To use the following Group Policy objects, you need the OneDrive sync build 18.1
...
+  2. Enable KFM Group Policy
```
Comment (JohanFreelancer9) [inline OneDrive/redirect-known-folders.md:99]:
```suggestion
  2. Enable KFM Group Policy.
```

File: OneDrive/redirect-known-folders.md [hunk 8]
```diff
@@ -87,3 +87,25 @@ To use the following Group Policy objects, you need the OneDrive sync build 18.1
...
+  3. Known folders should be moved successfully to OneDrive
```
Comment (JohanFreelancer9) [inline OneDrive/redirect-known-folders.md:100]:
```suggestion
  3. Known folders should have been moved successfully to OneDrive.
```

File: OneDrive/redirect-known-folders.md [hunk 9]
```diff
@@ -87,3 +87,25 @@ To use the following Group Policy objects, you need the OneDrive sync build 18.1
...
+- If folder(s) have been redirected to Network Share 
+
+  >[!NOTE]
```
Comment (JohanFreelancer9) [inline OneDrive/redirect-known-folders.md:103]:
```suggestion
  > [!NOTE]
```

File: OneDrive/redirect-known-folders.md [hunk 10]
```diff
@@ -87,3 +87,25 @@ To use the following Group Policy objects, you need the OneDrive sync build 18.1
...
+  > Windows 10 Fall Creators Update (version 1709 or later) or Windows Server 2019 and OneDrive build 17.3.7064.1005 or later recommended to get Files on Demand benefits
```
Comment (JohanFreelancer9) [inline OneDrive/redirect-known-folders.md:104]:
```suggestion
  > Windows 10 Fall Creators Update (version 1709 or later) or Windows Server 2019 and OneDrive build 17.3.7064.1005 or later are recommended to get Files on Demand benefits.
```

File: OneDrive/redirect-known-folders.md [hunk 11]
```diff
@@ -87,3 +87,25 @@ To use the following Group Policy objects, you need the OneDrive sync build 18.1
...
+  
+  1. Use a migration tool ([Mover](https://docs.microsoft.com/en-us/sharepointmigration/mover-fileshare-to-o365), [SharePoint Migration Tool](https://docs.microsoft.com/en-us/sharepointmigration/introducing-the-sharepoint-migration-tool), etc.) to copy contents in Network Share location to user's ODB, make sure all content goes into existing Documents, Pictures, or Desktop (create if it doesn’t exist) folders.
```
Comment (JohanFreelancer9) [inline OneDrive/redirect-known-folders.md:106]:
```suggestion
  1. Use a migration tool ([Mover](https://docs.microsoft.com/sharepointmigration/mover-fileshare-to-o365), [SharePoint Migration Tool](https://docs.microsoft.com/sharepointmigration/introducing-the-sharepoint-migration-tool), etc.) to copy contents in the Network Share location to a user's ODB, making sure that all contents go into existing Documents, Pictures, or Desktop (create if it doesn’t exist) folders.
```

File: OneDrive/redirect-known-folders.md [hunk 12]
```diff
@@ -87,3 +87,25 @@ To use the following Group Policy objects, you need the OneDrive sync build 18.1
...
+  2. Disable the Window Folder Redirection Group Policy, make sure to leave the folder and contents on OneDrive
```
Comment (JohanFreelancer9) [inline OneDrive/redirect-known-folders.md:107]:
```suggestion
  2. Disable the Window Folder Redirection Group Policy. Make sure to leave the folder and contents on OneDrive. 
```

File: OneDrive/redirect-known-folders.md [hunk 13]
```diff
@@ -87,3 +87,25 @@ To use the following Group Policy objects, you need the OneDrive sync build 18.1
...
+  2. Enable KFM Group Policy
+  3. Known folders should be moved successfully to OneDrive
+- If folder(s) have been redirected to Network Share 
+
+  >[!NOTE]
+  > Windows 10 Fall Creators Update (version 1709 or later) or Windows Server 2019 and OneDrive build 17.3.7064.1005 or later recommended to get Files on Demand benefits
+  
+  1. Use a migration tool ([Mover](https://docs.microsoft.com/en-us/sharepointmigration/mover-fileshare-to-o365), [SharePoint Migration Tool](https://docs.microsoft.com/en-us/sharepointmigration/introducing-the-sharepoint-migration-tool), etc.) to copy contents in Network Share location to user's ODB, make sure all content goes into existing Documents, Pictures, or Desktop (create if it doesn’t exist) folders.
+  2. Disable the Window Folder Redirection Group Policy, make sure to leave the folder and contents on OneDrive 
+  3. Enable KFM Group Policy
```
Comment (JohanFreelancer9) [inline OneDrive/redirect-known-folders.md:108]:
```suggestion
  3. Enable KFM Group Policy.
```

File: OneDrive/redirect-known-folders.md [hunk 14]
```diff
@@ -87,3 +87,25 @@ To use the following Group Policy objects, you need the OneDrive sync build 18.1
...
+  4. Known folders should be moved to OneDrive and will merge with the existing Desktop, Documents, and Pictures folders which should contain all of the Network share content that you moved in step b.
```
Comment (JohanFreelancer9) [inline OneDrive/redirect-known-folders.md:109]:
```suggestion
  4. Known folders should have been moved to OneDrive and will merge with the existing Desktop, Documents, and Pictures folders which should contain all of the Network share content that was moved in Step 2.
```

**Pull request merged**